Add application data replication

feddovanede opened the pull request:
Add application data replication

feddovanede committed:
Add application data replication

Add application data replication

MarkvanMents committed:
Proofread

MarkvanMents reviewed (approved):
Thanks for the PR,
Looks good. Please let me know when it can be published.

feddovanede committed:
Make licensed explicit

Make it explicit that the opt-out only affects licensed applications n Mendix Cloud

MarkvanMents reviewed (approved):
Changes look good. Let me know when I can publish.

MarkvanMents commented:
Approved over Slack.
